app/design/frontend/Belodubrovskyi/Theme/Magento_Newsletter/templates/subscribe.phtml

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+// @codingStandardsIgnoreFile
+
+/** @var \Magento\Newsletter\Block\Subscribe $block */
+
+?>
+<div class="block newsletter">
+    <?= $block->getChildHtml( 'title_text');?>
```

Comment:
Some commit names are confusing:
Lesson 2: Add text to the newsletter subscribe block - this commit does not add your text, it keeps the original code of the file Magento_Newsletter/templates/subscribe.phtml - which is correct, so the commit should have the name (Rewrite the template Magento_Newsletter/templates/subscribe.phtml 
Lesson 2: Fix git - has the line addition $block->getChildHtml( 'title_text');?> however the commit's name says nothing about this block
Please don't change anything now, but in future keep in mind the commit names and the changes which are made .